local.css

```diff
@@ -431,3 +431,9 @@ table.whitespace caption {
   background-color: #04AA6D;
   color: white;
 }
+
+td.exampleChar {
+  font-size: 36pt;
+  text-align: center;
+  vertical-align: center;
```

Comment:
This should probably be `vertical-align: middle;`. See https://w3c.github.io/csswg-drafts/css2/#propdef-vertical-align

Reply:
thanks. Fixed in 07cbc60.